nroam.el

```diff
@@ -121,7 +121,7 @@ Make the region inserted by BODY read-only, and marked with
 
 (defun nroam--org-roam-file-p ()
   "Return non-nil if the current buffer is an `org-roam' buffer."
-  (org-roam--org-roam-file-p))
+  (nroam--org-roam-file-p))
```

Comment:
this is introducing an infinite loop :-).

Reply:
:scream: whoops, I went a little too fast here... good catch, thanks.